src/openforms/conf/base.py

```diff
@@ -47,6 +47,14 @@
     ("nl", _("Dutch")),
 ]
 
+LANGUAGE_COOKIE_HTTPONLY = True
+
+LANGUAGE_COOKIE_NAME = "openforms_language"
+
+LANGUAGE_COOKIE_SAMESITE = "Lax"
```

Comment:
this needs to be `"None"` for cross-origin use, see the similar CSRF & Session cookie setings

Reply:
Ah okay, so whenever forms are embedded, you would want this to be "None"

Reply:
yep!